lib/exercism/stream_exercise.rb

```diff
@@ -2,13 +2,19 @@ module Stream
   Exercise = Struct.new(:id, :uuid, :problem, :last_activity, :last_activity_at, :iteration_count, :help_requested, :username, :avatar_url) do
     attr_writer :comment_count
 
+    #    def initialize(*args)
+    #      fail "Stream::Extercise constructed"
+    #      super(*args)
+    #    end
+
     def comment_count
       @comment_count || 0
     end
 
-    def at
-      @at ||= last_activity_at.to_datetime
-    end
+    # Unused? 20160801 1129
+    # def at
+    #  @at ||= last_activity_at.to_datetime
```

Comment:
No this is used. (in a view, and is not tested at all.)
:exclamation: do not merge :exclamation: